src/Pages/Join/Join.js

```diff
@@ -0,0 +1,354 @@
+            <img
+              className="joinFormImg"
+              src="https://www.t2tea.com/on/demandware.static/-/Library-Sites-T2SharedLibrary/default/dwb93811dd/images/content/tea-society-offer.png"
+              alt="formImg"
+            />
+          </div>
+          <form className="formTable">
+            <div className="nameTable">
+              <div className="firstNameWrap">
+                <div
+                  className={`firstName ${
+                    formErrors.firstName.length > 0 ? "error" : null
+                  }`}
+                >
+                  <input
+                    onChange={this.handleChange}
+                    type="text"
+                    placeholder="First name"
+                    name="firstName"
+                  />
+                </div>
+                {formErrors.firstName.length > 0 && (
+                  <span className="errorMessage">{formErrors.firstName}</span>
+                )}
+              </div>
+              <div className="lastNameWrap">
+                <div
+                  className={`lastName ${
+                    formErrors.lastName.length > 0 ? "error" : null
+                  }`}
+                >
+                  <input
+                    onChange={this.handleChange}
+                    type="text"
+                    placeholder="Last name"
+                    name="lastName"
+                  />
+                </div>
+                {formErrors.lastName.length > 0 && (
+                  <span className="errorMessage">{formErrors.lastName}</span>
+                )}
+              </div>
+            </div>
+            <div className="inputFormSub">
+              <div
+                className={`emailWrapper ${
+                  formErrors.email.length > 0 ? "error" : null
+                }`}
+              >
+                <input
+                  onChange={this.handleChange}
+                  className="loginEmail"
+                  type="text"
+                  placeholder="Email*"
+                  name="email"
+                />
+              </div>
+              {formErrors.email.length > 0 && (
+                <span className="errorMessage">{formErrors.email}</span>
+              )}
+              <div
+                className={`phoneWrapper ${
+                  formErrors.phone.length > 0 ? "error" : null
+                }`}
+              >
+                <input
+                  onChange={this.handleChange}
+                  className="phoneInput"
+                  type="text"
+                  placeholder="Contact number*"
+                  name="contactNumver"
+                />
+              </div>
+              {formErrors.phone.length > 0 && (
+                <span className="errorMessage">{formErrors.phone}</span>
+              )}
+
+              <div
+                className={`pwdInputWrapper ${
+                  formErrors.password.length > 0 ? "error" : null
```

Comment:
```suggestion
                  formErrors.password.length ? "error" : ""
```
이건 어떨까요?
- null이 리턴됐을 때 "pwdInputWrapper null" <- 이렇게 입력되겠네요
- 길이는 정수, 0 초과하는 값은 무조건 true로 처리되겠죠?